### DISCUSSION TITLE
Philosophy of Elements - by NawrasDima

### DISCUSSION THREAD

**Discussion by NawrasDima** [planetarium/NCIPs#19] [Ideas]
7 days ago, I shared this suggestion [ [Link](https://discord.com/channels/928926944937013338/1211565839321530428/1267767465526689955) ] in [Planetarium Dev Server](https://discord.gg/nDJTBdAz) then one of 9C devs (Seungmin) asked me if I can post this for you guys to discuss it further and answer any question you have. So here I am, and this is how it goes:
{
Hello Devs,
I am a huge fan of Web3 Gaming especially Mobile ones and I wanted to share with you a Suggestion if you may have it:
1- To make the game -kinda- more realistic, we should be adding the 5th Element of Life, the #Light ! 
Also, I noticed there is an Element called "Normal" which is odd knowing that normally it doesn't exist in the philosophy of Elements!
2-  (A) Wind is a sacred element, immune element, the moving power of all which has a descending dominance over: B- Land/ground 🟤, C- Water 🔵 D- Fire 🔴 E- Light 🟡 , respectively. On the other hand, other elements (e.g. Water) can only dominate the next in order element (= Fire ❤️‍🔥).

3- A Light world could be developed to extend the game based on ( 1-).

4- Light 🟡 have no dominance over anything, but when it is combined with Water 🔵 they give Life 🟢 or by another meaning Healing/Regeneration power.

5- ☀️ + 🫧 = 🌲 concept
Crafting items 🙄 logically should be following the above concept.
For example:
Instead of spending Golden dust to get Ymir, you could also get it by combining equal tier materials of Water 🔵+ Wind ⚪ = Frost 🥶 Ymir (with 0.1% success rate e.g) ... etc but if the cat is wearing any Fire ❤️‍🔥item, Ymir's effectiveness will drop dramatically! ☝️

6- I have plenty of many ideas to expand 9C philosophy of elements, characters, combining items, mechanism, story telling, special events... etc but I want to make sure if the above ones make any sense first 😅 
So I can proceed.

Thank you for your reading and feedback.

Best of luck Devs 🫡:red_fukindown:
}
End.
![image](https://github.com/user-attachments/assets/05034159-0e62-4781-bb92-a07b9619618a)

**Comment by boscohyun** [comment]
Congratulations on your first discussion!🥳

**Reply by NawrasDima** [reply]
Yeah! Thank you
